drivers/scheduler/k8s/specs/fio-fstrim/fio-config-map.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fio-job-config
+data:
+    fio.job: |
+        [global]
+        name=fio-rand-RW
+        directory=/scratch/
+        rw=randwrite
+        randrepeat=1
+        rwmixread=25
```

Comment:
If the workflow is for fstrim. Why do we need to reads here? We can save time by doing writes and deletes.

Reply:
Addressed